tests/models.py

```diff
@@ -1352,3 +1353,89 @@ def test_operator_clear(self):
         self.assertEqual(ti2.try_number, 1)
         # try_number (0) + retries(1)
         self.assertEqual(ti2.max_tries, 1)
+
+    def test_xcom_disable_pickle_type(self):
```

Comment:
Should we test that switching the flag after objects are already stored results in desirable state